sdk/tests/resources/js-test-pre.js

```diff
@@ -179,6 +190,57 @@ function testFailed(msg)
     _flushBufferedLogsToConsole();
 }
 
+/**
+ * Sets the current test name for usage within testPassedOptions/testFailedOptions.
+ * @param {string} name The name to set as the current test name.
+ */
+function setCurrentTestName(name)
+{
+	this._currentTestName = name;
```

Comment:
You can't rely on having a valid "this" pointer here. Just define _currentTestName in the global scope in this file and assign to it, and remove the definition in the anonymous function above.